src/libraries/Common/src/System/Net/Security/Unix/SafeDeleteContext.cs

```diff
@@ -29,15 +29,24 @@ protected SafeDeleteContext(SafeFreeCredentials credential)
             _credential.DangerousAddRef(ref ignore);
         }
 
+        public SafeDeleteContext(IntPtr handle) : base(handle, true)
+        {
+            _credential = null;
+        }
+
+        public SafeDeleteContext(IntPtr handle, bool ownsHandle) : base(handle, ownsHandle)
+        {
+            _credential = null;
```

Comment:
Are either of these `_credential = null` assignments necessary?  The default value will be null.

Reply:
no. I did deeper cut and removed `_credential`  completely so we don't drag unneeded property.